tests/cypress/e2e/features/consensus.js

```diff
@@ -64,20 +65,161 @@ context('Basic manipulations with consensus job replicas', () => {
+            cy.mergeConsensusJob(400);
+            cy.get('.cvat-notification-notice-consensus-merge-task-failed')
+                .should('be.visible')
+                .invoke('text').should('include', 'Could not merge the job');
+            cy.closeNotification('.cvat-notification-notice-consensus-merge-task-failed');
 
-            // Check asc order of jobs
+            // Check asc order of jobs and save job ids
             function parseJobId(jobItem) {
                 const jobItemText = jobItem.innerText;
                 const [start, stop] = [0, jobItemText.indexOf('\n')];
                 return +(jobItemText.substring(start, stop).split('#')[1]);
             }
             cy.get('.cvat-job-item').then((jobItems) => {
                 const sourceJobId = parseJobId(jobItems[0]);
+                jobIDs.push(sourceJobId);
                 for (let i = 1; i <= replicas; i++) {
                     const jobId = parseJobId(jobItems[i]);
+                    jobIDs.push(jobId);
                     expect(jobId).equals(sourceJobId + i);
                 }
             });
         });
+
+        it('Check consensus management page', () => {
+            const defaultQuorum = 50;
+            const defaultIoU = 40;
+            cy.contains('button', 'Actions').click();
+            cy.contains('Consensus management').should('be.visible').click();
+            cy.get('.cvat-consensus-management-inner').should('be.visible');
+            // Save settings, confirm request is sent
+            let requestCount = 0;
+            cy.intercept('PATCH', 'api/consensus/settings/**', () => {
+                requestCount++;
+            }).as('settingsMeta');
+            cy.contains('button', 'Save').click();
+            cy.wait('@settingsMeta');
+            cy.get('.ant-notification-notice-message')
+                .should('be.visible')
+                .invoke('text')
+                .should('eq', 'Settings have been updated');
+            cy.closeNotification('.ant-notification-notice-closable');
+
+            // Forms and invalid saving
+            cy.get('#quorum').then(([$el]) => {
+                cy.wrap($el).invoke('val').should('eq', `${defaultQuorum}`);
+                cy.wrap($el).clear();
+            });
+            cy.get('.ant-form-item-explain-error').should('be.visible');
+            cy.contains('button', 'Save').click();
+            cy.get('#iouThreshold').then(([$el]) => {
+                cy.wrap($el).invoke('val').should('eq', `${defaultIoU}`);
+                cy.wrap($el).clear();
+            });
+            cy.get('.ant-form-item-explain-error').should('be.visible');
+            cy.contains('button', 'Save').click();
+            cy.then(() => {
+                expect(requestCount).to.equal(1);
+            });
+            cy.get('.ant-notification-notice').should('not.exist');
+
+            // Go back to task page
+            cy.get('.ant-btn-default').should('be.visible').click();
```

Comment:
Lets add a class inside `<GoBackButton />` component so we dont adress it with `.ant-btn-default`